### PR TITLE
Add log file to GitHub issue output

### LINKED ISSUE #9398: Fatal errors should include the path to the post-mortem INFO log
Currently, we provide this information, which no one sees: https://minikube.sigs.k8s.io/docs/handbook/troubleshooting/#post-mortem-minikube-debug-logs

I was thinking that when we have a fatal exit, we should be able to say something like:

```
Please open a GitHub issue. If you are able to drag and drop the following log-file into the issue, we'll be able to make faster progress: /path/to/minikube.INFO.log
```

Related: #9323

### PR DESCRIPTION
**Pull request by spowelljr**
Closes #9398

Adds a line to the output when we tell the user to create a GitHub issue. The line contains the path of the latest log file to be attached to the GitHub issue to aid in debugging the issue.

Tested on Mac and Linux so far, will test on Windows as well.

Before
```
❌  Exiting due to GUEST_START: wait 6m0s for node: waiting for apps_running: expected k8s-apps: missing components: kube-dns

😿  If the above advice does not help, please let us know: 
👉  https://github.com/kubernetes/minikube/issues/new/choose
```

After
```
❌  Exiting due to GUEST_START: wait 6m0s for node: waiting for apps_running: expected k8s-apps: missing components: kube-dns

╭────────────────────────────────────────────────────────────────────╮
│                                                                    │
│    😿  If the above advice does not help, please let us know:      │
│    👉  https://github.com/kubernetes/minikube/issues/new/choose    │
│                                                                    │
│    Please attach the following file to the GitHub issue:           │
│    - /Users/powellsteven/.minikube/logs/lastStart.txt              │
│                                                                    │
╰────────────────────────────────────────────────────────────────────╯
```

### CODE REVIEW

**Review by medyagh — COMMENTED** [review]
please add before after this PR to the description

File: pkg/minikube/out/out.go [hunk 1]
```diff
@@ -359,6 +360,29 @@ func DisplayError(msg string, err error) {
...
+	}
+	ErrT(style.Tip, "If you are able to drag and drop the following log-file into the issue, we'll be able to make faster progress: {{.logFileName}}", V{"logFileName": logFileName})
+}
+
+func getLatestLogFileName() (string, error) {
+	dir := os.Getenv("TMPDIR")
+	if dir == "" {
+		dir = "/tmp/"
+	}
+	args := fmt.Sprintf("cd %s && echo %s$(ls -t *minikube_*_*_*log | head -1)", dir, dir)
```
Comment (medyagh) [inline pkg/minikube/out/out.go:376]: this wont work on windows.

how about we point them to the log file we store in the minikube home folder ? 

ideally if there is a trick to Open the Issue with that file attached would best

**Review by medyagh — CHANGES_REQUESTED** [review]

File: pkg/minikube/out/out.go (hunk 1 above)
Comment (spowelljr) [reply pkg/minikube/out/out.go:376]: We could just always point them to the start file, but what if `minikube addon enable ingress` fails? Should we still point them to the start logs?

Comment (spowelljr) [reply pkg/minikube/out/out.go:376]: Also, I don't think there is anyway to create a URL that will attach the file. Sounds like that would be a large security threat if it was a thing.

Comment (medyagh) [reply pkg/minikube/out/out.go:376]: good point, how about we detect what command is failing, if it is not start, we will look for latest temp files.

regardless,  this command wont work windows, can you make it a Go way of finding latest temp files that has word minikube in their file name?

**Review by medyagh — CHANGES_REQUESTED** [review]

File: pkg/minikube/out/out.go (hunk 1 above)
Comment (spowelljr) [reply pkg/minikube/out/out.go:376]: > how about we detect what command is failing, if it is not start, we will look for latest temp files
Can do that for sure

> can you make it a Go way of finding latest temp files
Will look into how to do that

**Review by medyagh — CHANGES_REQUESTED** [review]
also change UI to be more easy to see 
maybe use the Red Box ui helper we have

```
❌  Exiting due to GUEST_START: wait 6m0s for node: waiting for apps_running: expected k8s-apps: missing components: kube-dns

😿  If the above advice does not help, please let us know by filing 
----- Red Box thing ----- ----- ----- ----- -----  --
👉  https://github.com/kubernetes/minikube/issues/new/choose

   Drag and rop the following files into a github issue...
    -  /Users/powellsteven/.minikube/logs/lastStart.txt
----- ----- ----- ----- ----- ----- ----- 
```

File: pkg/minikube/out/out.go (hunk 1 above)
Comment (spowelljr) [reply pkg/minikube/out/out.go]: Done

**Comment by medyagh** [comment]
please update the UI in the PR description whenever new UI ready

**Review by medyagh — CHANGES_REQUESTED** [review]
lets add back the crying cat emoji
`😿 `

**Review by medyagh — APPROVED** [review]

**Pull request merged**